sdk/sdk-api/src/main/java/mil/emp3/json/geoJson/GeoJsonExporter.java

```diff
@@ -208,7 +228,16 @@ private void appendRectangle(final Rectangle feature, StringBuffer buffer) throw
                 feature.getGeoId().toString(), feature.getName(), feature.getDescription(),
                 "PBS_RECTANGLE--", coordinateStr, altitudeModeStr, scale, boundingBoxStr,
                 modifiers, attributes, RENDER_JSON, 0);
-        buffer.append(geoJSON);
+        try {
```

Comment:
This block seems to be repeated quite a bit. Condense to method if possible.

Reply:
done